week-1/G-strings-methods/exercise2.js

```diff
@@ -1,3 +1,4 @@
-var name = " Daniel  ";
+var name = " Ermiyas!  ";
+var message= name + "Hello mates";
 
 console.log(message);
```

Comment:
Nice!
But when you run this exercise, it prints: " Ermiyas!  Hello mates". How can you change it to say something more meaninfgul? More of a greeting or introducing yourself? :)